components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -3363,9 +3363,12 @@ public boolean doUpdateGroup(Group oldGroup, Group newGroup) throws CharonExcept
                 usernameClaimUri = getPrimaryLoginIdentifierClaim();
             }
 
+            UserStoreManager userStoreManager = carbonUM.getSecondaryUserStoreManager(userStoreDomainOfUser);
             // Check if the user ids & associated user name sent in updated (new) group exist in the user store.
             String userId =
-                    carbonUM.getUserIDFromProperties(usernameClaimUri, userName, UserCoreConstants.DEFAULT_PROFILE);
```

Comment:
check for null to avoid possible null pointers

Reply:
Fixed in https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/377/commits/d2bc6d9757159b2a4c21408b61381eccb46dd62e